css-dev/burf-theme/layout/_navigation.scss

```diff
@@ -868,7 +869,9 @@ $_search-height: 103px;
 	}
 
 	&.nav-open {
-		.content {
+		.bu-banner,
+		.content
+		 {
 			@include breakpoint( $left-nav-desktop ) {
 				left: $width-side-nav - 60px;
```

Comment:
Is the 60px here referring to the width of the search button? If so, it'd be great to use `$nav-toggle-size` here! https://github.com/bu-ist/responsive-foundation/blob/develop/css-dev/burf-theme/layout/_navigation.scss#L384

Reply:
Not sure what the 60px is referring to, it was there to begin with for the .content selector. I just added the .bu-banner selector to it.

I don't think it refers to the search button width because this chunk of code pushes the banner and content to the right when the sidenav is toggled open.

Reply:
Ah! Got it. Then it's probably best to leave it alone. That makes sense.